common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -52,7 +52,7 @@ class SequenceFields(object):
             "Note, you must enable Entrance Exams for this course setting to take effect."
         ),
         default=False,
-        scope=Scope.content,
+        scope=Scope.settings,
```

Comment:
I feel like we went back and forth on how to scope this parameter during the initial feature buildout and decided on Scope.content for a particular reason.  However that reason escapes me now.